### PR TITLE
MultilineWhitespaceBeforeSemicolonsFixer - support static calls

### PR DESCRIPTION
**Pull request by ntzm**
Ref #3529 

Not really a bug as it was never supported, so targetting master

Just duplicated all the `$foo->` tests and changed them to `Foo::`, let me know if you want any other cases added

### CODE REVIEW

**Comment by keradus** [comment]
Note:
target shall be the same as it will be decided in
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3478

**Comment by egircys** [comment]
Thank you @ntzm !

File: tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php [hunk 1]
```diff
@@ -424,6 +499,265 @@ public function provideSemicolonForChainedCallsFixCases()
...
+                ?>',
+                '<?php
+
+                    Service
+                        ::method2();
+                ?>',
+            ], [
+                '<?php
+
+                    Service::method1()
```
Comment (egircys) [inline tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php]: Should it support also multiple static calls?
```php
Service::method1()
    ::method2()
    ->method3()
;
```

Comment (ntzm) [reply tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php]: What would `method1` return to allow that?

Comment (ntzm) [reply tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php]: Ah like something that returns a class string, I think it should be allowed but I want to know what others think

Comment (egircys) [reply tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php]: Yes, something like this.. not sure if there are any real-life cases for it.
```php
<?php

class A 
{
    public static function fa()
    {
        return 'B';
    }
}

class B
{
    public static function fb()
    {
        return new C();
    }
}

class C
{
    public function fc()
    {
        return 'c';
    }
}

echo A::fa()
    ::fb()
    ->fc();

```

Comment (ntzm) [reply tests/Fixer/Semicolon/MultilineWhitespaceBeforeSemicolonsFixerTest.php]: Added some tests for chained static calls, thanks @egircys

**Comment by keradus** [comment]
Travis is failing on PHP 5.6.
Could you take a look @ntzm ?

**Comment by keradus** [comment]
Thank you @ntzm.

**Pull request merged**